capstone-frontend/src/components/AdminClubPage.jsx

```diff
@@ -83,10 +82,9 @@ class AdminClubPage extends Component {
       endDateTime: new Date(document.getElementById('end-datetime').value).getTime(), 
       attendeeEmails: this.state.members.map(m => m.email),
       organizerEmail: JSON.parse(window.localStorage.getItem('profileObj')).email, 
-    }
+    };
     fetch('/api/meetings', {method: 'post', body: JSON.stringify(meeting)})
-        .then(response => response.json())
-        .then(json => console.log(json))
+        .then(history.push(`/clubpage/${meeting.clubID}`))
```

Comment:
Same here: this navigation will happen before the POST completes if it isn't wrapped in a function